vaadin-notification-flow-demo/pom.xml

```diff
@@ -104,7 +115,7 @@
                     <plugin>
                         <groupId>org.eclipse.jetty</groupId>
                         <artifactId>jetty-maven-plugin</artifactId>
-                        <configuration>
+                        <configuration combine.self="override">
```

Comment:
Nit: The command `mvn -pl vaadin-notification-flow-demo -Pwar install jetty:run` from `README` is not working anymore. Only running directly from `demo` folder.
Should it be updated in `README`?

Reply:
Weirdly it seems to work without the `install`... Updated readme.